ui/src/app/sidebar/sidebar.scss

```diff
@@ -51,7 +52,7 @@ $deselected-text: #818d94;
     &__version {
         cursor: pointer;
         width: 100%;
-        text-align: center;
+        text-align: left;
```

Comment:
I noticed that if the version is even longer, the collapsed button will get pushed to the right and will not be in view.

Try changing the css for version to this:

    &__version {
        cursor: pointer;
        width: 110px;
        text-align: left;
        white-space: nowrap;
        overflow: hidden;
        text-overflow: ellipsis;
    }

and see if it is ok with you.

Everything else LGTM

Reply:
great suggestion, thank you!